lib/stat_tracker.rb

```diff
@@ -493,4 +493,30 @@ def best_season(team_id)
 
       return "#{winningest_season}#{winningest_season.next}"
   end
+
+  def worst_season(team_id)
+    team_seasons = Hash.new { |season_id, games_lost| season_id[games_lost] = [0.0, 0.0] }
+    season = CSV.open(@game_teams_path, headers: true, header_converters: :symbol)
+    season.each do |row|
+      if row[:team_id] == team_id
+        team_seasons[row[:game_id][0..3]][0] += 1
+        if row[:result] == 'LOSS'
+          team_seasons[row[:game_id][0..3]][1] += 1
+        end
+      end
+    end
+    highest_loss_percentage = 0.0
```

Comment:
it was interesting that when we were trying to find the highest loss percentage it was erroring. but when we switched to lowest win percentage we were able to find the correct solution. im still not sure why that is the case

Reply:
i realize i made this comment on the wrong commit. this comment is for the latest commit of the pull request. not for the earliest of this pull request. sorry